session/database.rst

```diff
@@ -11,6 +11,64 @@ across different servers.
 Symfony can store sessions in all kinds of databases (relational, NoSQL and
 key-value) but recommends key-value databases like Redis to get best performance.
 
+Define Session Storage by DSN
+-----------------------------
+
+Besides specifying a service as the session-handler you can also provide a
+"DSN" directly and let Symfony create the necessary services/clients:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/framework.yaml
+        framework:
+            session:
+                # ...
+                handler_id: 'redis://password@redis-server:6379'
+
+    .. code-block:: xml
+
+        <!-- config/packages/framework.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:framework="http://symfony.com/schema/dic/symfony"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/symfony https://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+            <framework:config>
+                <framework:session enabled="true" handler-id="redis://password@redis-server:6379"/>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/framework.php
+        $container->loadFromExtension('framework', [
+            'session' => [
+                // ...
+                'handler_id' => 'redis://password@redis-server:6379',
+            ],
+        ]);
+
+Supported DSN protocols are:
```

Comment:
I would move this part in a `.. note::` directive under the first example if this section were to be removed.